### ISSUE TITLE
"val_loss" doesn't converge

### ISSUE THREAD

**Issue by FlorPeng** [daniilidis-group/EvAC3D#2]
# Summary
Hello,

I executed the publicly available EvAC3D training code on the "coffee2" subset of the MOEC-3D dataset. Unfortunately, it appears that the validation loss fails to converge. Upon inspection of the TensorBoard logs, it seems that the loss significantly increases approximately every 1800 steps.

What steps can be taken to address and improve this issue? Any insights or suggestions would be greatly appreciated.

![image](https://github.com/daniilidis-group/EvAC3D/assets/79638033/1ec37a79-57d6-4f5d-9f52-9d361ff5ad5b)

# How to reproduce
`python ./training_code/train.py`
- The event data and the mask data are "coffee2" from [MOEC-3D](https://drive.google.com/drive/u/1/folders/1frNjCATu2I2KAcvfPMHctmLVeGTP9nmg).
- All hyperparameters are left as default. 
- The same event file is used for both train and test data.

![image](https://github.com/daniilidis-group/EvAC3D/assets/79638033/92c87358-7ace-4196-a4e1-9e13ce331b04)

# Misc
- Train loss converged properly.
![image](https://github.com/daniilidis-group/EvAC3D/assets/79638033/64c49a98-56af-426f-b6d8-8a58ad232ed0)
